src/Utilities/VariableTemplates/VariableTemplates.jl

```diff
@@ -117,7 +117,7 @@ end
       offset += N
     elseif T <: StaticArray
       N = length(T)
-      retexpr = :(array[$(offset + 1):$(offset + N)] .= val)
+      retexpr = :(array[$(offset + 1):$(offset + N)] .= val[:])
```

Comment:
Is this to fix shape mismatch with the symmetric matrix?

Reply:
No, if you have a `SMatrix` in the `Vars`

Reply:
Everywhere right now we have `SVector`